library/modules/bigip_wait.py

```diff
@@ -134,6 +141,12 @@ def timeout(self):
             return None
         return int(self._values['timeout'])
 
+    @property
```

Comment:
remove not needed, you have already specified defaults in the arg spec

Reply:
Removed